src/message_dao.py

```diff
@@ -109,6 +109,15 @@ def get(self, message_id: str) -> Optional[Message]:
         msg.children = self._get_children(message_id)
         return msg
 
+    @lru_cache()
```

Comment:
hmm, what benefit does caching give us?
I don't imagine we'd call this function multiple times for the same exact message.

Reply:
People do *sometimes* upload a new revision within hours of an earlier revision, but I am with @dlatypov, I don't think that happens frequently enough to pay for the overhead of an lru_cache, let alone make it worthwhile.

Reply:
That's true, I can't imagine many cases where cache helps out so I've removed it.